crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -5138,7 +5143,14 @@ impl<'db> TypeInferenceBuilder<'db> {
     ) -> Result<Type<'db>, CompareUnsupportedError<'db>> {
         let db = self.db();
 
-        let contains_dunder = right.class().class_member(db, "__contains__").symbol;
+        let contains_dunder = right
```

Comment:
This should use `try_call_dunder_get` instead -- can we add a TODO for that?

Reply:
we already have an issue open for it here FWIW https://github.com/astral-sh/ruff/issues/16371